hathor/storage/rocksdb_storage.py

```diff
@@ -69,3 +69,6 @@ def get_or_create_column_family(self, cf_name: bytes) -> 'rocksdb.ColumnFamilyHa
         if cf is None:
             cf = self._db.create_column_family(cf_name, rocksdb.ColumnFamilyOptions())
         return cf
+
+    def close(self) -> None:
+        del self._db
```

Comment:
This could be `self._db.close()` instead. I'm testing to see if it works for me as well.

Reply:
Just did it. Thanks!